components/organisms/Filters/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, PropsWithChildren } from "react";
+import classNames from "classnames";
+import Container, {
+  type ContainerProps,
+} from "@rubin-epo/epo-react-lib/Container";
+import Search from "@/components/molecules/Search";
+import ClearFiltersButton from "@/components/molecules/ClearFiltersButton";
+import styles from "./styles.module.scss";
+
+interface FiltersProps {
+  className?: string;
+  hasSearch?: boolean;
+  width?: ContainerProps["width"];
+  onFiltersCleared?: () => void;
+}
+
+const Filters: FC<PropsWithChildren<FiltersProps>> = ({
```

Comment:
re-usable filters bar that is pretty much the same `FilterBar` but without the bespoke logic of the datalist pages